src/server.py

```diff
@@ -74,9 +64,9 @@ def parse_request(request):
 
 
 def resolve_uri(uri):
+    """Handle URI request and return appropriate result"""
     req_path = RSC.format(uri.decode('utf8'))
     try:
-        #pdb.set_trace()
         if os.path.isdir(req_path):
```

Comment:
notice that RSC is a relative path.  This means that your server *must* be running in the directory that holds `webroot`.  It might be nice to have an absolute path to webroot so you could have the server running elsewhere.